presto-product-tests/src/main/java/com/facebook/presto/tests/cli/PrestoCliTests.java

```diff
@@ -43,6 +43,14 @@
         extends PrestoCliLauncher
         implements RequirementsProvider
 {
+    @Inject
+    @Named("databases.presto.server_address")
+    protected String serverAddress;
```

Comment:
Thinking about it some more, it would be better to use tabases.presto.cli_server_address here.  We can set the appropriate value for the profile in the yaml files.